modules/test/playwright/tests/locked-items-web/locked-items-web.spec.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {lockedItemsPagesTest} from './fixtures/lockedItemsPagesTest';
+
+const test = mergeTests(
+	loginTest,
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPD-11003': true,
+		'LPS-180328': true,
+	}),
+	lockedItemsPagesTest
+);
+
+const testFeatureFlagDisabled = mergeTests(
+	loginTest,
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPD-11003': false,
+	}),
+	productMenuPageTest
+);
+
+testFeatureFlagDisabled(
+	'the Locked Items item does not show in the Product Menu when FF is not enabled',
+	async ({productMenuPage}) => {
+		await productMenuPage.goToConfiguration();
+		await expect(productMenuPage.lockedItemsMenuItem).toBeHidden();
+	}
+);
```

Comment:
I'm not sure if we need to test it, looks confusing if we don't need to test the opposite but I don't have enough context :(

Reply:
I see your point, but I think is OK testing this. I suppose we can delete this test once the FF is removed.

Reply:
I feel like a superfluous test that we must clean in the future, I don't see much value here but maybe I don't have enough context.

Reply:
The only value is that we don't incorrectly show something not supposed to be shown. It feels right to me. As I said, it should be deleted when FF is removed.